README.md

```diff
@@ -77,6 +77,12 @@ python -m pip install "featuretools[nlp]"
 python -m pip install "featuretools[tsfresh]"
 ```
 
+**Dask Support** - Use Dask Dataframes to create EntitySets or run DFS with njobs > 1
+
+```
+python -m pip install "featuretools[dask]"
```

Comment:
Also add to docs:
https://github.com/alteryx/featuretools/blob/main/docs/source/install.md